### DISCUSSION TITLE
404 Error when trying to publish w/ GitHub!

### DISCUSSION THREAD

**Discussion by Karan-Ghatt** [community/community#23733] [Other Feature Feedback, Questions, & Ideas]
<p>I’m trying to host a page on GitHub pages, but I seem to be having some errors with it. From what I understand, everything is where it should be, but I keep getting a 404 error when I try and access the page.</p>
<p>Link to the repo: <a href="https://github.com/Karan-Ghatt/My-Portfolio" rel="nofollow noopener">https://github.com/Karan-Ghatt/My-Portfolio</a></p>
<p>It should be active at: <a href="https://karan-ghatt.github.io/My-Portfolio/" rel="nofollow noopener">https://karan-ghatt.github.io/My-Portfolio/</a></p>
<p>I would really appreciate if you could have a look and see if you can tell what’s going on. Much love people!</p>
<p>[EDIT]<br>
I managed to get it loading! I had saved my file as Index.html, where it should have been index.html. All sorted on that front now! But I cant seem to get all the images to load! It manages to find the about image but nothing else, even though their in the same directory in the repo??</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/karan-ghatt">@Karan-Ghatt</a>, I just tried the link that you provided and your site loads for me. Are you still having issues loading it yourself?</p>

**Comment by Karan-Ghatt** [comment]
<p>I managed to get it loading! I had saved my file as Index.html, where it should have been index.html. All sorted on that front now! But I cant seem to get all the images to load!  It manages to find the about image but nothing else, even though their in the same directory in the repo??</p>

**Comment by ThirdEyeRose** [comment]
<p>Do you have an example of an image link that is breaking? I don’t see any broken images.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Take a look at the difference in how you write the <code>src</code> value. This one works:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/Karan-Ghatt/My-Portfolio/blob/e9c8c585f51ce71e04e775e7572d990782c8f11f/index.html#L160" target="_blank">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/Karan-Ghatt/My-Portfolio/blob/e9c8c585f51ce71e04e775e7572d990782c8f11f/index.html#L160" target="_blank">Karan-Ghatt/My-Portfolio/blob/e9c8c585f51ce71e04e775e7572d990782c8f11f/index.html#L160</a></h4>
<pre class="onebox"><code class="lang-html"><ol class="start lines" start="150" style="counter-reset: li-counter 149 ;">
<li>    &lt;h6 data-aos="fade-left" data-aos-delay="150"&gt;Veiw Recent Projects&lt;/h6&gt;
</li>
<li>&lt;/div&gt;
</li>
<li>&lt;/div&gt;
</li>
<li>
</li>
<li>&lt;div class="row" data-aos="fade-down" data-aos-delay="200"&gt; 
</li>
<li>
</li>
<li>    &lt;!-- Portfolio Item 1 | AB Removals --&gt;
</li>
<li>    &lt;div class="column"&gt;
</li>
<li>        &lt;div class="portfolio-container"&gt;
</li>
<li>            &lt;div class="item" id="1"&gt;
</li>
<li class="selected">                &lt;img src="./images/abremovals.png" style="width:100%"  alt=""&gt;
</li>
<li>                &lt;div class="text"&gt;
</li>
<li>                    &lt;h3&gt;PROJECT 1&lt;/h4&gt;
</li>
<li>                        &lt;p&gt;A|B Home Removals&lt;/p&gt;
</li>
<li>                    &lt;/div&gt;
</li>
<li>                    &lt;div class="button"&gt;Learn More&lt;/div&gt;
</li>
<li>                &lt;/div&gt;
</li>
<li>        &lt;/div&gt;
</li>
<li>
</li>
<li>        &lt;!-- Portfolio Item 2 | Resume --&gt;
</li>
<li>        &lt;div class="portfolio-container"&gt;
</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>This one doesn’t:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/Karan-Ghatt/My-Portfolio/blob/e9c8c585f51ce71e04e775e7572d990782c8f11f/index.html#L239" target="_blank">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/Karan-Ghatt/My-Portfolio/blob/e9c8c585f51ce71e04e775e7572d990782c8f11f/index.html#L239" target="_blank">Karan-Ghatt/My-Portfolio/blob/e9c8c585f51ce71e04e775e7572d990782c8f11f/index.html#L239</a></h4>
<pre class="onebox"><code class="lang-html"><ol class="start lines" start="229" style="counter-reset: li-counter 228 ;">
<li>                &lt;/div&gt;
</li>
<li>                &lt;div class="button"&gt;Learn More&lt;/div&gt;
</li>
<li>            &lt;/div&gt;
</li>
<li>        &lt;/div&gt;
</li>
<li>&lt;/div&gt;
</li>
<li>
</li>
<li>            &lt;!-- Portfolio Item 7 | Password Bank --&gt;
</li>
<li>            &lt;div class="column"&gt;
</li>
<li>                &lt;div class="portfolio-container"&gt;
</li>
<li>                    &lt;div class="item" id="7"&gt;
</li>
<li class="selected">                        &lt;img src="/images/passwordvault.png" style="width:100%" alt=""&gt;
</li>
<li>                        &lt;div class="text"&gt;
</li>
<li>                            &lt;h3&gt;PROJECT 7&lt;/h4&gt;
</li>
<li>                                &lt;p&gt;Password Vault&lt;/p&gt;
</li>
<li>                            &lt;/div&gt;
</li>
<li>                            &lt;div class="button"&gt;Learn More&lt;/div&gt;
</li>
<li>                        &lt;/div&gt;
</li>
<li>                    &lt;/div&gt;
</li>
<li>                    &lt;!-- Portfolio Item 8 | Instagram Bot --&gt;
</li>
<li>                    &lt;div class="portfolio-container"&gt;
</li>
<li>                    &lt;div class="item" id="8"&gt;
</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>The reason for this is that the second one, <code>/images/passwordvault.png</code>, means the same thing as <code>https://karan-ghatt.github.io/images/passwordvault.png</code>. Whereas, if you wrote it as <code>./images/passwordvault.png</code> or <code>images/passwordvault.png</code> it would be the same thing as writing <code>https://karan-ghatt.github.io/My-Portfolio/images/passwordvault.png</code>, which would work.</p>
<p>You can learn more about <a href="https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#A_quick_primer_on_URLs_and_paths">how URLs and hyperlinks work in web pages on the Mozilla Developer Network documentation</a>.</p>
